lib/src/backend/test_platform.dart

```diff
@@ -92,3 +85,35 @@ class TestPlatform {
 
   String toString() => name;
 }
+
+List<TestPlatform> _all = [
```

Comment:
I'm told that since all the values are TestPlatform the reified type will already be correct.